### PR TITLE
Feature/key value app

### PR DESCRIPTION
**Pull request by jdmedlock**
Add Key Value app specification

### CODE REVIEW

File: Projects/Key-Value-App.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+a key is pressed.
+
+## Useful links and resources
+
+- [Digital Electronics](https://en.wikipedia.org/wiki/Digital_electronics)
+- [Keyboard Event](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent)
+
+## Example projects
+
+[Keycode](https://www.linkedin.com/feed/update/urn:li:activity:6519081404398075904/)
```
Comment (florinpop17) [inline Projects/Key-Value-App.md]: Broken link?

Comment (jdmedlock) [reply Projects/Key-Value-App.md]: This link works for me. What symptom are you seeing? If necessary I can switch to a different example such as --> https://unixpapa.com/js/testkey.html

Comment (florinpop17) [reply Projects/Key-Value-App.md]: Maybe it’s because you linked a linkedin link and it’s only visible on your profile.
Yeah, you could change it

Comment (jdmedlock) [reply Projects/Key-Value-App.md]: Updated and merged

Comment (florinpop17) [reply Projects/Key-Value-App.md]: Perfect! Thanks @jdmedlock !

**Pull request merged**